.circleci/config.yml

```diff
@@ -3,48 +3,55 @@ version: 2.1
 orbs:
   cri-api:
     executors:
+      python_3_9:
+        docker:
+          - image: cimg/python:3.9
       python_3_8:
         docker:
-          - image: circleci/python:3.8
+          - image: cimg/python:3.8
       python_3_7:
         docker:
-          - image: circleci/python:3.7
-      python_3_6:
-        docker:
-          - image: circleci/python:3.6
+          - image: cimg/python:3.7
     jobs:
+      lint:
+        parameters:
+          python-version:
+            type: string
+        executor: python_<< parameters.python-version >>
+        steps:
+          - checkout
+          - run:
+              name: Install pip dependencies
+              command: pipenv install --dev --deploy --python=$(which python)
+          - run:
+              name: Ensure code is correctly formatted
+              command: pipenv run black --check .
+
```

Comment:
minor nit: there's a space between the `lint` job and this one but not on the others.